sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -651,50 +651,52 @@ object QueryExecutionErrors {
 
   def missingJdbcTableNameAndQueryError(
       jdbcTableName: String, jdbcQueryString: String): Throwable = {
-    new IllegalArgumentException(
-      s"Option '$jdbcTableName' or '$jdbcQueryString' is required."
-    )
+    new SparkIllegalArgumentException(errorClass = "MISSING_JDBC_TABLE_NAME_AND_QUERY",
+      messageParameters = Array(jdbcTableName, jdbcQueryString))
   }
 
   def emptyOptionError(optionName: String): Throwable = {
-    new IllegalArgumentException(s"Option `$optionName` can not be empty.")
+    new SparkIllegalArgumentException(errorClass = "EMPTY_OPTION",
+      messageParameters = Array(optionName))
   }
 
   def invalidJdbcTxnIsolationLevelError(jdbcTxnIsolationLevel: String, value: String): Throwable = {
-    new IllegalArgumentException(
-      s"Invalid value `$value` for parameter `$jdbcTxnIsolationLevel`. This can be " +
-        "`NONE`, `READ_UNCOMMITTED`, `READ_COMMITTED`, `REPEATABLE_READ` or `SERIALIZABLE`.")
+    new SparkIllegalArgumentException(errorClass = "INVALID_JDBC_TXN_ISOLATION_LEVEL",
```

Comment:
When the class name is updated, it should be updated here as well: `INVALID_JDBC_TRANSACTION_ISOLATION_LEVEL`